doc/release/CHANGES.txt

```diff
@@ -31,6 +31,7 @@ E  473	Several modules are not included in build when JDK11 is used
 E  493	javamail.providers file missing from provider jars after 1.6.5
 E  505	WriteTimeoutSocket::getFileDescriptor$ support for Conscrypt
 E  512	Improve code coverage in logging-mailhandler tests
+E  528	CompactFormatter precision and surrogate pairs
```

Comment:
this needs to go to 2.0.2 version. 2.0.1 is on review for almost 3 weeks already , see https://github.com/jakartaee/specifications/pull/339

Reply:
Added a new section for 2.0.2 release to the changes.txt.